src/components/courseCard/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Rating from '../rating/index';
+
+import styles from './style.module.scss';
+
+import { links } from '../../utils/constants';
+import Img from '../img/index';
+
+export default function CourseCard({ item }) {
+  const { title, author, rate, price, img = '', id = 1, votes } = item;
+
```

Comment:
Import order + prop-types + export const(`export function` in your case)

Reply:
Done